src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1152,14 +1322,18 @@ query_path(const char *szInput, int *is_target_path, dfs_obj_t **parent, char *i
 					if (rc)
 						DL_WARN(rc, "daos_eq_create() failed");
 					main_eqh = td_eqh;
-					rc       = pthread_atfork(NULL, NULL, &child_hdlr);
-					D_ASSERT(rc == 0);
 				}
 				D_MUTEX_UNLOCK(&lock_eqh);
 			}
 
 			atomic_store_relaxed(&daos_inited, true);
 			atomic_fetch_add_relaxed(&daos_init_cnt, 1);
+
+			bool rc_cmp_swap;
```

Comment:
Please don't mix delcarations and code, this should go on L1293

Reply:
The declaration is relocated as you suggested. Thank you!